### PR TITLE
[Community Tutorial][Edit Existing] Add hyperlink to image source in Beginner's Cookbook tutorial

### PR DESCRIPTION
**Pull request by TC25**
Updates to the url for this tutorial.

### CODE REVIEW

File: tutorials/beginners-cookbook/index.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ region.
 
 ![Raster](map-algebra.png)
 
-Image sources: https://gisgeography.com/spatial-data-types-vector-raster/
+[Image sources](https://gisgeography.com/spatial-data-types-vector-raster/)
```
Comment (jdbcode) [inline tutorials/beginners-cookbook/index.md:82]: How about making is a little more explicit?:

\_Conceptual figures sourced from \[GISGeography\]\(https://gisgeography.com/spatial-data-types-vector-raster/)_

**Review by jdbcode — CHANGES_REQUESTED** [review]
Thanks for breaking this out into its own PR!

One suggestion to make a little more explicit.

File: tutorials/beginners-cookbook/index.md (hunk 1 above)
Comment (TC25) [reply tutorials/beginners-cookbook/index.md]: Done

**Review by jdbcode — APPROVED** [review]
LGTM

**Comment by jdbcode** [comment]
Thanks for the changes, TC! The [page](https://developers.google.com/earth-engine/tutorials/community/beginners-cookbook) should be updated within the week.

**Comment by TC25** [comment]
Thanks!

**Review by jdbcode — APPROVED** [review]

**Comment by jdbcode** [comment]
Changes are live: https://developers.google.com/earth-engine/tutorials/community/beginners-cookbook

Thanks, TC!

**Comment by TC25** [comment]
Thank you!

**Pull request merged**